tests/datasource/fluent/integration/test_integration_datasource.py

```diff
@@ -168,7 +168,7 @@ def test_sql_query_data_asset(empty_data_context):
                     "..", "..", "..", "test_sets", "taxi_yellow_tripdata_samples"
                 )
             ),
-            None,
+            r".*",
```

Comment:
Wasn't this meant to test the default behavior?

Reply:
But if we are providing it explicitly it is not the default. I understand why you had to though.

Reply:
Yeah, because of how the test is setup, I could add a conditional or pass it as kwargs.